index.html

```diff
@@ -146,175 +108,158 @@ <h2 class="section-title">Speakers</h2>
-              In this talk, I present ‘secateur’, a tool for Twitter users to
-              protect themselves from some forms of online harassment. I
-              discuss using Python and Django to create tools that integrate
-              with Twitter, use its APIs, and block lots of people (LOTS of
-              people).
-            </p>
           </li>
 
         </ul>
 
       </div>
     </div>
--->
 
-<!--
+
+
     <div class="tickets">
       <div class="wrap">
 
         <h2 class="section-title">Tickets</h2>
 
         <ul class="ticket-options group">
-
+          <p align="center">Tickets to PyConline AU include access to all talks, including DjangoCon AU<br><br><BR></p>
           <li class="option group">
-            <span class="name">DjangoCon AU Only</span>
-            <p class="description">Specialist Track Day ticket. Does not include PyConline AU.</p>
-            <span class="price"><span class="currency">AU&#36;</span>150</span>
+            <span class="name">PyConline AU Student</span>
+            <p class="description">Admission for students currently enrolled in full or part-time education, at any level. Conditions apply.</p>
+            <span class="price"><span class="currency">AU&#36;</span>10</span>
           </li>
 
           <li class="option group">
-            <span class="name">PyConline AU Enthusiast + DjangoCon AU</span>
-            <p class="description">Discounted rates for enthusiasts paying out of their own pocket.</p>
-            <span class="price"><span class="currency">AU&#36;</span>490</span>
+            <span class="name">PyConline AU Enthusiast</span>
+            <p class="description">General admission for Python users attending the conference.</p>
+            <span class="price"><span class="currency">AU&#36;</span>29</span>
           </li>
 
           <li class="option group">
             <span class="name">PyConline AU Professional</span>
-            <p class="description">If your company is paying, or you can claim as a tax deduction. Includes DjangoCon AU.</p>
-            <span class="price"><span class="currency">AU&#36;</span>690</span>
+            <p class="description">If your employer is paying for your ticket, or you make your living using Python, please consider this tier.</p>
+            <span class="price"><span class="currency">AU&#36;</span>79</span>
           </li>
 
           <li class="option group">
             <span class="name">PyConline AU Contributor</span>
-            <p class="description">Give a little extra to help with financial aid for other attendees. Includes DjangoCon AU.</p>
-            <span class="price"><span class="currency">AU&#36;</span>990</span>
+            <p class="description">All funds raised by contributor tickets are used to support outreach and diversity in future years.</p>
+            <span class="price"><span class="currency">AU&#36;</span>300+</span>
           </li>
 
         </ul>
```

Comment:
The link below in line 268 is wrong: Needs to be https://2020.pycon.org.au/attend/ or https://pretix.eu/pyconau/online2020/